### ISSUE TITLE
Add "Skillup Attempts" Field

### ISSUE THREAD

**Issue by FaxCelestis** [madcowfred/padherder-issues#546]
Add a "Skillup Attempts" field in the Update Monster Info frame, and add mouseover to the Skill Level displayed on the main table showing skillup percentage rate.

For instance, I kept track of skillups on my Leviathan over the course of Princess Punt. I'd like to be able to put "101" into an "Attempted Skillups" field in Leviathan's info box, and when I mouseover his now-maxed skill, have it display "Skill-Up Rate: 22/101 (21.78%)"

**Comment by bloodwars** [comment]
the skillup rate is locked at 10% though, 20% w/ 2x, so i'm not sure that that'd really be worth the effort (imho)

**Comment by FaxCelestis** [comment]
I mostly want to know what kind of returns I'm getting on feeding rates. During Princess Punt, for instance, I went 22/101 (21.78%) on Leviathan skillups but only 14/105 (13.33%) on D'spinas skillups, and I went 5/9 (55.55%) on Cinderella skillups during Angry Birds, but am currently rocking a 1/31 (3.22%) on Echidna skillups. Right now I'm using the comments field, but it'd be nice to have it do it for me.
